docker/Dockerfile

```diff
@@ -0,0 +1,4 @@
+# Same image as in docker-compose.yaml
+FROM akeneo/pim-php-base:4.0
+COPY --from=composer:1 /usr/bin/composer /usr/local/bin/composer
```

Comment:
Oh, that's an interesting way to do that!

Good stuff keying `composer:1`. Thats quick to miss.